Tribler/dispersy

```diff
@@ -1 +1 @@
-Subproject commit d056b388836cec227f6752683045534325b5b5e4
+Subproject commit ab39486c5572c1e613ea5d8a5fcac9d61d56f802
```

Comment:
Sure, go back to the right commit, in the submodule and commit. When you squash everything together, it will be gone.